docs/business.yaml

```diff
@@ -4649,9 +4649,14 @@ components:
           - foreignJurisdiction
           - authorization
           - nameRequest
+          - contactPoint
           - offices
           - parties
+          - shareStructure
           properties:
+            isApproved:
+              type: boolean
+              description: 'Continuation In Authorization has been approved.'
```

Comment:
We don't need to publish `isApproved`. Its only used for internal purpose (for validation)

Reply:
Removed. The only change I see is authorization date is not required anymore and added some required properties